src/client/img/home-icons/account-creation-fraud.svg

```diff
@@ -0,0 +1,75 @@
+<svg width="42" height="42" viewBox="0 0 42 42" fill="none" xmlns="http://www.w3.org/2000/svg">
+<rect x="1" y="1" width="40" height="40" rx="7.5" fill="white"/>
+<rect x="1" y="1" width="40" height="40" rx="7.5" fill="url(#paint0_linear_2219_39281)" fill-opacity="0.02"/>
+<rect x="1" y="1" width="40" height="40" rx="7.5" stroke="#F0F0ED" stroke-width="1.2" stroke-linecap="round"/>
+<g filter="url(#filter0_i_2219_39281)">
+<g filter="url(#filter1_i_2219_39281)">
+<path d="M13.6665 28.4999C13.6664 27.2169 14.0366 25.961 14.7326 24.8832C15.4286 23.8053 16.4209 22.9511 17.5903 22.4233C18.7598 21.8954 20.0567 21.7162 21.3255 21.9072C22.5942 22.0981 23.7809 22.6512 24.7432 23.4999" stroke="#F35B22" stroke-width="1.4" stroke-linecap="round" stroke-linejoin="round"/>
+</g>
+<g filter="url(#filter2_i_2219_39281)">
```

Comment:
As discussed on Slack, these filters automatically exported from Figma cause blurring issues in Safari, let's remove them.